### DISCUSSION TITLE
why My account has been blocked for suspected abuse. Please contact Support

### DISCUSSION THREAD

**Discussion by J-W-Chan** [community/community#139089] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I didn't do anything
![image](https://github.com/user-attachments/assets/e0178233-f6ce-40ca-ae6d-8ecad564a1e8)

**Comment by SavinRazvan** [comment]
Did you try to penetrate the model? What inputs did you had?

**Reply by J-W-Chan** [reply]
I just passed the whitelist application and haven't done anything

**Reply by anuxs** [reply]
+1 me too

**Reply by anuxs** [reply]
>>Your access to Models/Copilot has been restricted because our systems detected the use of a credential sharing service which modifies and interferes with how Copilot works.

this is my return

**Reply by J-W-Chan** [reply]
> You can submit feedback by using the **Give Feedback** option available on the page: [GitHub Marketplace Models](https://github.com/marketplace/models).
> 
> * If your claim is valid and not a result of a false positive from the models, the ban should be lifted upon review.
> * **Please provide a detailed, step-by-step procedural outline to replicate the testing process**, allowing for the verification of any potential false positives in the model.

This discussion was initiated by me with feedback, but I couldn't find a place to submit it

**Comment by CNSeniorious000** [comment]
> Your account has been blocked for suspected abuse. Please contact Support

+1 me too

I've opened a ticket on GitHub Support about 2 months ago but haven't received any response.

**Comment by SavinRazvan** [comment]
**From what I know, models still send a lot of `false flags`, and you should ask for help here: https://support.github.com/.** 
A similar thing happened with OpenAI’s models, and people were in the same situation when the initial versions were released.

Make sure after they lift your bans to post here the result to help other people too, maybe mark the response as answer too.

**Comment by KateCatlin** [comment]
Hey all! Sorry for the trouble with hitting our blockers here! I know we've looked into this a bit and hopefully have it resolved by now. Did you all get access back? Did the support process not get you unblocked in any cases? If this is still an issue, we can reopen the investigation and look into it more. Thanks in advance!

**Reply by KateCatlin** [reply]
No one has chimed back in since this post, so I'm going to assume this issue is resolved and close this issue! Please reopen it if you have an issues with access, we're happy to help :)
